crates/red_knot_python_semantic/resources/mdtest/expression/boolean.md

```diff
@@ -133,9 +133,9 @@ if NotBoolable():
 ```py
 def test(cond: bool):
     class NotBoolable:
-        __bool__ = None if cond else 3
+        __bool__: int | None = None if cond else 3
 
-    # error: [unsupported-bool-conversion] "Boolean conversion is unsupported for type `NotBoolable`; it incorrectly implements `__bool__`"
+    # error: [unsupported-bool-conversion] "Boolean conversion is unsupported for type `NotBoolable`; its `__bool__` method isn't callable"
```

Comment:
Here `__bool__` is a union, but unlike above, both elements are non-callable, so we treat the union as a whole as non-callable.